src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.report;
+
+import core.basesyntax.storage.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Remove the empty line at the start of the class implementation.